### PR TITLE
Adapt ioc dockerfile to new machine-application and mathphys install targets

### PR DESCRIPTION
**Pull request by anacso17**
This PR changes Dockerfile.iocs to use new repositories install targets.

I think it might be interesting to tag this repository before and after this PR, because the changes here imply different installation procedures for previous deploys. What do you think?

### CODE REVIEW

**Comment by fernandohds564** [comment]
I think it is a good idea @anacso17!

**Review by fernandohds564 — APPROVED** [review]

File: dockerfile-templates/Dockerfile.iocs [hunk 1]
```diff
@@ -38,6 +38,6 @@ RUN cd repos && \
     git clone https://github.com/lnls-sirius/machine-applications && \
     cd machine-applications/ && \
     git checkout :pkg_version_machine_applications && \
-    make install-scripts
+    make install
 
 ENV EPICS_CA_ADDR_LIST="10.128.1.12:5064 10.128.1.12:5070 10.128.1.54:5064 10.128.1.55:5064 10.128.1.13:5064 10.128.1.13:5068 10.128.1.13:5070 10.0.38.29:5064 10.0.38.37:5064 10.0.38.39:5064 10.0.38.92:5064 10.0.38.93:5064"
```
Comment (xresende) [inline dockerfile-templates/Dockerfile.iocs:43]: @anacso17 , maybe we should check if this ADDR_LIST needs updating...

**Review by xresende — APPROVED** [review]

**Comment by xresende** [comment]
> I think it is a good idea @anacso17!



> I think it is a good idea @anacso17!

@anacso17 , I agree too.

File: dockerfile-templates/Dockerfile.iocs (hunk 1 above)
Comment (anacso17) [reply dockerfile-templates/Dockerfile.iocs]: done!

**Review by xresende — APPROVED** [review]

**Review by fernandohds564 — APPROVED** [review]

**Pull request merged**